src/main/java/seedu/duke/ui/Ui.java

```diff
@@ -49,4 +49,12 @@ public static void printRouteMessage(ArrayList<Bus> options) {
             System.out.println("none.\n\tThere are no direct bus routes. :(");
         }
     }
+
+    public static void printAllBusMessage(ArrayList<Bus> allBuses) {
+        System.out.println("\nThe buses available in NUS are: ");
+        for (Bus bus : allBuses) {
+            System.out.println(bus.toString());
```

Comment:
Are you printing all buses and their specific routes together?

Reply:
Yes, currently it will show 
![Untitled](https://user-images.githubusercontent.com/56810260/95769370-5b629500-0cea-11eb-8872-8bb5bb5c3bf7.png)